lisa/schema.py

```diff
@@ -454,6 +454,12 @@ class DiskOptionSettings(FeatureSettings):
             decoder=partial(search_space.decode_set_space_by_type, base_type=DiskType)
         ),
     )
+    os_disk_size: search_space.CountSpace = field(
```

Comment:
Make sure `_call_requirement_method`, `__eq__`, `__repr__`, `__get_key` proceeded the new field.

Reply:
Done